### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by lyubenkov**

### CODE REVIEW

File: utils/__init__.py [hunk 1]
```diff
@@ -15,3 +16,18 @@ def create_allure_environment_opts(opts: dict, dst: tp.Optional[pathlib.Path] =
...
         )
         file.write("\n")
+
+
+def setup_logging(log_level=logging.DEBUG):
+    """Setup root logger and quiet some levels."""
+    logger = logging.getLogger()
+    logger.setLevel(log_level)
+
+    logging.getLogger("web3.RequestManager").setLevel(logging.WARNING)
```
Comment (gigimon) [inline utils/__init__.py:28]: pls add comment why you change this level for web3

File: integration/tests/basic/erc/test_ERC721.py [hunk 2]
```diff
@@ -113,14 +112,18 @@ def test_symbol(self, erc721):
         assert symbol == "MPL"
 
     def test_balanceOf(self, erc721):
-        balance_before = erc721.contract.functions.balanceOf(erc721.account.address).call()
+        with allure.step("Get balance for user"):
```
Comment (gigimon) [inline integration/tests/basic/erc/test_ERC721.py]: this additional allure.step doesn't provide additional information without context. If will a problem we will understand where without this steps.

File: integration/tests/basic/erc/test_ERC721.py [hunk 3]
```diff
@@ -421,27 +436,30 @@ def test_getApproved_incorrect_token(self, erc721):
...
 
         erc721.transfer_solana_from(erc721.account.address, solana_address, token_id, erc721.account)
         opts = TokenAccountOpts(token_mint)
 
-        wait_condition(
-            lambda: int(
-                sol_client.get_token_accounts_by_owner_json_parsed(acc.public_key, opts)
-                .value[0]
-                .account.data.parsed["info"]["tokenAmount"]["amount"]
+        with allure.step("Get token data and token amount"):
```
Comment (gigimon) [inline integration/tests/basic/erc/test_ERC721.py]: pls remove allure.steps from this test

**Review by gigimon — APPROVED** [review]
just remove additional with allure.step from erc721

File: utils/__init__.py (hunk 1 above)
Comment (lyubenkov) [reply utils/__init__.py:28]: Added

File: integration/tests/basic/erc/test_ERC721.py (hunk 2 above)
Comment (lyubenkov) [reply integration/tests/basic/erc/test_ERC721.py]: Following the task goal: "Add more debug information for big cases (erc721 for example, add manually additional info) in the context of allure" - this step adds information to allure report test body, and we can see now in test body that first we got balance, second we did mint, and lastly we check balance again. Before that test body in allure report was just empty.

However as asked, I removed them.

File: integration/tests/basic/erc/test_ERC721.py (hunk 3 above)
Comment (lyubenkov) [reply integration/tests/basic/erc/test_ERC721.py]: Removed.

**Comment by lyubenkov** [comment]
> just remove additional with allure.step from erc721

As asked, removed all allure steps from erc721.

File: integration/tests/basic/erc/test_ERC721.py (hunk 2 above)
Comment (gigimon) [reply integration/tests/basic/erc/test_ERC721.py]: Yes, more debugging. But just comment and step without additional info (for example balances, tx, user id) is useless. Just remove and make assert more clear (but you already)

**Review by kristinaNikolaevaa — APPROVED** [review]

**Pull request merged**